lib/tasks/green_lanes.rake

```diff
@@ -121,20 +121,17 @@ namespace :green_lanes do
     # load existing data
     theme = GreenLanes::Theme.where(section: '1', subsection: '3').first
     assessments = GreenLanes::CategoryAssessment.all.index_by do |assessment|
-      [assessment.measure_type_id, assessment.regulation_id]
+      [assessment.measure_type_id, assessment.regulation_id, assessment.regulation_role]
     end
-    types = %w[551 552 553 554 555 561 562 564 565 566 570 695]
+    types = %w[551 552 553 554 555 561 562 564 565 566 570 690 695 696]
 
     GreenLanes::CategoryAssessment.db.transaction do
-      current_date = Time.zone.today
-      actual_condition = Sequel.lit('validity_start_date <= ? AND ' \
-                                      '(validity_end_date IS NULL OR validity_end_date > ?)')
-
-      Measure.where(actual_condition, current_date, current_date)
-             .where(measure_type_id: types, measure_generating_regulation_role: 1)
+      # select active measure_id, regulation_id and regulation_role of given measure type to create category_assessment
+      Measure.where(measure_type_id: types)
```

Comment:
I think you'll want to wrap this in `TimeMachine.now do ... end`

Reply:
now it set with the thread current provided by active support, isn't it?
here - 
https://github.com/trade-tariff/trade-tariff-backend/blob/5f91a668f475cd2790b04ae2d21e75f46911963a/app/lib/trade_tariff_request.rb#L3